cmake/CMakeSettings.json

```diff
@@ -0,0 +1,62 @@
+{
+  "configurations": [
+    {
+      "name": "WinML-Debug",
```

Comment:
These configurations are manually set with build.py right? Why do we need these?

Reply:
Native VS integration with CMake